work-with-hash-set

ap2144839 opened the pull request:
- creating class com.peterson.collection.WorkWithHashSet
- changing class com.peterson.util.Util
https://trello.com/c/gqSV1SGR/6-work-with-hashset

ap2144839 committed:
work-with-hash-set
- creating class com.peterson.collection.WorkWithHashSet
- changing class com.peterson.util.Util
https://trello.com/c/gqSV1SGR/6-work-with-hashset

Alexis220382 commented on src/com/peterson/collection/WorkWithHashSet.java:
```diff
@@ -0,0 +1,21 @@
+            hashSet.add(i);
+        }
+        return hashSet;
+    }
```
Где-то я такой же кусок кода уже видел)))
Копипаст, батенька, злейший враг.

Alexis220382 commented on src/com/peterson/util/Util.java:
```diff
@@ -152,6 +152,74 @@ public static void main(String args[]){
+        stop = System.currentTimeMillis();
+        System.out.println("Deleting first element from HashSet of 1,000,000 elements takes: " + (stop-start) + " ms");
+
     }
```
Вот эту `агромную партянку` тоже можно зарефачить, чтоб выглядела читаемо)))

ap2144839 committed:
work-with-hash-set
- refactoring class com.peterson.util.Util
- class com.peterson.collection.WorkWithHashSet has been deleted;
https://trello.com/c/gqSV1SGR/6-work-with-hashset
